AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/FirebaseTestLabController.kt

```diff
@@ -111,7 +111,8 @@ internal class FirebaseTestLabController(
         pullScreenshots: Boolean = false,
         cachedTestMatrixFilter: CachedTestMatrixFilter,
         testTargets: List<String>? = null,
-        flakyTestAttempts: Int = 2
+        flakyTestAttempts: Int = 2,
+        testTimeout: String = "2700s"
```

Comment:
Should we take the timeout as an Int and then only convert it to a string at the every end?

Reply:
Done.

Reply:
Actually, can you name this argument testTimeoutSeconds so units are obvious.